holamundo.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Hello World</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
+    <script>
+        //los objetos de JS son navigator, Document, Window y history
+      document.write("Hello World");
+      alert("Hello World 2");
+      console.log ("Hello World 3");
+    </script>
+</head>
+<body>
+</body>
+</html>
```

Comment:
Always add EOF lines to files to prevent issues in the long term. You can configure your editor using editorconfig.